test/test_complex.py

```diff
@@ -18,6 +19,22 @@ def fn(a: complex):
         ''')
         self.assertEqual(b, cu.fn(b))
 
+    def test_pickle(self):
+        class ComplexModule(torch.nn.Module):
+            def __init__(self):
+                super(ComplexModule, self).__init__()
+                self.a = 3 + 5j
+
+            def forward(self, b: int):
+                return b
+
+        scripted = torch.jit.script(ComplexModule())
+        buffer = io.BytesIO()
+        torch.jit.save(scripted, buffer)
+        buffer.seek(0)
+        loaded = torch.jit.load(buffer)
```

Comment:
I think you can use `self.getExportImportCopy` here, which will save and load from a buffer for you.